.github/workflows/browser-tests.yml

```diff
@@ -57,11 +62,16 @@ jobs:
 
             - name: Checkout PR branch
               uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
 
             - name: Setup Job and Install Dependencies
               uses: ./.github/actions/setup-job
 
             - uses: actions/download-artifact@v4
+              with:
+                  pattern: tachometer-results-*
```

Comment:
Being more explicit here to ensure we are only downloading the assets we require.